connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -67,6 +67,8 @@
     protected static final String SYNC_TOPIC_ACLS = "sync.topic.acls";
     protected static final String EMIT_HEARTBEATS = "emit.heartbeats";
     protected static final String EMIT_CHECKPOINTS = "emit.checkpoints";
+    protected static final String AUTH_OFFSET_RESET = "auto.offset.reset";
+    protected static final String ENABLE_AUTO_COMMIT = "enable.auto.commit";
```

Comment:
Can we reuse `ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG` instead of defining a new constant?